content/800-guides/330-management-api-basic.mdx

```diff
@@ -0,0 +1,160 @@
+
+
+:::tip OpenApi
+The API reference is available via an [OpenAPI 3.1. spec](https://api.prisma.io/v0/swagger-editor).
+:::
+
+## Prerequisites
+
+- Node.js and `npm` installed
+- A [Prisma Console](https://cloud.prisma.io/) account
+
+## 1. Create a service token in Prisma Console
+
+First, you need to create a service token to be able to access your workspace in Prisma Console:
+
+1. Open the [Prisma Console](https://cloud.prisma.io/)
+2. Navigate to the **Integrations** page of your workspace
+3. Click **Generate integration token**
+4. Copy and save the generated service token securely
+
+## 2. Set up your project directory
+
+### 2.1. Create a basic TypeScript project
+
+Open your terminal and run the following commands:
+
+```terminal
+# Create a new directory and navigate into it
+mkdir management-api-demo
+cd management-api-demo
+```
+
+Next, initialize npm and install dependencies required for using TypeScript:
+
+```terminal
+# Initialize a new npm project
+npm init -Y
+
+# Install dependencies
+npm install tsx typescript @types/node --save-dev
+
+# Create the necessary files
+touch index.ts
+```
+
+You now have an `index.ts` file that you can execute with `npx tsx index.ts`. It's still empty, we'll start writing code in step 3.
+
+### 2.2. Configure service token environment variable
+
+Create you `.env` file:
+
+```terminal
+touch .env
+```
+
+Next, install the [`dotenv`](https://github.com/motdotla/dotenv) library for loading environment variables from the `.env` file:
+
+```terminal
+npm install dotenv
+```
+
+Finally, add your service token (from step 1) to `.env`:
+
+```bash
+PRISMA_SERVICE_TOKEN="ey..."
+```
+
+### 2.3. Install the `axios` library for HTTP request
+
+You're going to use [`axios`](https://github.com/axios/axios/tree/main) as your HTTP client to interact with the Management API. Install it as follows:
+
+```terminal
+npm install axios
+```
+
+You're all set, let's write some code to create a project and provision a Prisma Postgres database!
+
+## 3. Programmatically create a new project with a database
+
+Paste the following code into `index.ts`:
+
+```ts
+import axios from 'axios';
+import dotenv from 'dotenv';
+
+// Load environment variables
+dotenv.config();
+
+const API_URL = 'https://api.prisma.io';
+const SERVICE_TOKEN = process.env.PRISMA_SERVICE_TOKEN;
+
+if (!SERVICE_TOKEN) {
+  throw new Error('PRISMA_SERVICE_TOKEN is not set in the environment');
+}
+
+// Set HTTP headers to be used in this script
+const headers = {
+  Authorization: `Bearer ${SERVICE_TOKEN}`,
+  'Content-Type': 'application/json',
+};
+
+async function main() {
+  // Create a new project in your Prisma Console workspace
+  const projectName = `demo-project-${Date.now()}`;
```

Comment:
Isn't this a database name, vs. project name? 🤔

Reply:
It actually ends up being both (because the default DB is named after the project) but by design it's designated to be the name of the project you're creating in your workspace (that's why it's used as `name` when accessing the `POST /project` endpoint):

```
  const projectName = `demo-project-${Date.now()}`;
  const region = 'us-east-1';
  const createProjectRes = await axios.post(
    `${API_URL}/projects`,
    { name: projectName, region },
    { headers }
  );
```